spec/menu_spec.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/menu'
+
+describe Menu do
+
+  let(:subject) { Menu.new(nil)}
+
+  describe '#initialize' do
+    it 'initializes with the menu' do
+      expect(subject.menu).to include('Hallumi fritter burger' => 8)
```

Comment:
I like the use of to include. That's better than how I would have thought to do it.